.github/workflows/gh-ci-cron.yaml

```diff
@@ -32,21 +32,20 @@ jobs:
       with:
         os-type: "ubuntu"
 
-    - name: setup_miniconda
-      uses: conda-incubator/setup-miniconda@v2
+    - name: setup_micromamba
+      uses: mamba-org/provision-with-micromamba@main
       with:
-        python-version: "3.11"
-        miniforge-variant: Mambaforge
-        miniforge-version: latest
-        channel-priority: strict
-        channels: conda-forge, bioconda
-        add-pip-as-python-dependency: true
-        architecture: x64
+        environment-file: False
+        environment-name: mda
+        extra-specs: |
+          python=="3.11"
+          pip
+        channels: jaimergp/label/unsupported-cudatoolkit-shim, conda-forge, bioconda
```

Comment:
Question what needs `jaimergp/label/unsupported-cudatoolkit-shim`?

Reply:
OpenMM otherwise we're pulling down cuda everywhere. - it's quite widely used everywhere nowadays

Reply:
Cool beans